src/ensembl/production/metadata/api/genome.py

```diff
@@ -340,3 +340,46 @@ def fetch_genomes_info(
                 yield [{'genome': genome, 'datasets': dataset}]
         except Exception as e:
             raise ValueError(str(e))
+
+    def get_popular_organisms_counts(self, release_version=None):
+        from ensembl.production.metadata.api.models.organism import OrganismGroup, Organism, OrganismGroupMember
+        from sqlalchemy.orm import aliased
+        from sqlalchemy import func
+        with self.metadata_db.session_scope() as session:
+            session.expire_on_commit = False
+            o_species = aliased(Organism)
+            o = aliased(Organism)
+            if not release_version:
+                # Get latest released organisms
+                query = session.query(
```

Comment:
We were also having more descriptive variable for storing the query than query. Not a big deal, just inconsistent.